### ISSUE TITLE
Training dataset

### ISSUE THREAD

**Issue by hs11015** [MediaBrain-SJTU/MVFA-AD#4]
Hello, I have read your paper thoroughly and found it very informative. After reading it, I became interested and tried running the code myself. However, I couldn't find information about the dataset used for training. When I downloaded the provided dataset, I noticed that only the validation and test sets were included. After counting, I realized that the 'Train (with labels)' mentioned in Table 6 in the appendix corresponds to the good and Ungood data in the valid directory, and the 'Test' set is indeed the test directory. However, there is no information provided about the 'Train (all-normal)' data.
Could you please provide more information about this dataset or let me know if I need to request it separately? I would appreciate your assistance. Thank you.

**Comment by chaoqinhuang** [comment]
This work focuses on few-shot AD, aiming to utilize a small amount of data to get generalizable anomaly detection models. Therefore, there is no need to use the 'Train (all-normal)' large-scale training dataset, which is typically required for only the full-shot baselines like MKD and PatchCore. You could refer to Line 98 in [dataset](https://github.com/MediaBrain-SJTU/MVFA-AD/tree/main/dataset/medical_few.py) to see how we use the valid data. If you are still interested in accessing the 'Train (all-normal)' training dataset, you can follow the [BMAD](https://github.com/DorisBao/BMAD) to download the relevant datasets.

**Comment by hs11015** [comment]
Thank you very much for your kind response. After hearing your explanation, it's clear to me that the reason for providing the 'Train (all-normal)' information in Table 6 of the paper was to provide additional context for the full-normal-shot method mentioned in Table 1. I had misunderstood Table 6 in the appendix, thinking that you used both 'Train (all-normal)' and 'Train (with labels)' to obtain the results in Table 1, which led to my question. I came across your paper while researching medical diagnostic studies, and I'm delighted to see research that demonstrates generalizable performance across modalities and anatomical regions in the medical domain. It's promising to see the potential for diagnosis using AI. I will download and explore the BMAD all-normal dataset you provided for future research. Thank you once again for your valuable research and for providing this information.
